tests/test_queries.py

```diff
@@ -819,6 +837,80 @@ def validate(self, expected_values=None):
+    NAME = "dataset_mapping_plan"
+
+    def __init__(self, handle_data_skew, persist, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._handle_data_skew = handle_data_skew
+        self._persist = persist
+
+    def make_query(self) -> None:
+        self._model = ProjectQueryModel(
+            name=self.NAME,
+            project=ProjectQueryParamsModel(
+                project_id=self.get_project_id(),
+                include_dsgrid_dataset_components=False,
+                dataset=DatasetModel(
+                    dataset_id="tempo_mapped",
+                    source_datasets=[
+                        StandaloneDatasetModel(dataset_id="tempo_conus_2022"),
+                    ],
+                ),
+                mapping_plan=[
+                    DatasetMappingPlan(
+                        dataset_id="tempo_conus_2022",
+                        mappings=[
+                            MapDimensionOperation(
+                                dimension_name="county",
+                                handle_data_skew=self._handle_data_skew,
+                                persist=self._persist,
+                            ),
+                            MapDimensionOperation(
+                                dimension_name="model_year",
+                                handle_data_skew=self._handle_data_skew,
+                                persist=self._persist,
+                            ),
+                            MapDimensionOperation(
+                                dimension_name="end_use",
+                                handle_data_skew=self._handle_data_skew,
+                                persist=self._persist,
+                            ),
+                            MapDimensionOperation(
+                                dimension_name="subsector",
+                                handle_data_skew=self._handle_data_skew,
+                                persist=self._persist,
+                            ),
+                        ],
+                    ),
+                ],
+            ),
+            result=QueryResultParamsModel(
+                aggregations=[
+                    AggregationModel(
+                        dimensions=DimensionNamesModel(
+                            geography=["state"],
+                            metric=["end_uses_by_fuel_type"],
+                            model_year=[],
+                            scenario=[],
+                            sector=[],
+                            subsector=[],
+                            time=[],
+                            weather_year=[],
+                        ),
+                        aggregation_function="sum",
+                    ),
+                ],
+                output_format="parquet",
+                table_format=UnpivotedTableFormatModel(),
+            ),
+        )
+        return self._model
+
+    def validate(self, expected_values=None) -> bool:
```

Comment:
A few questions about these tests:
- I assume this test is working on a small version of tempo_conus_2022? 
- Can we test some expected_values?
- Should we also verify that mapping in a different order doesn't change the result (which would also require comparing to expected_values)?
- Would it be worthwhile to test that the journal is logging dimension names in the expected order?